assignments/objects.js

```diff
@@ -11,45 +11,86 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: "Example",
+  email: "examples@you.edu",
+  gender: "F"
+};
 
 // Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
+const mitzi = {
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+};
+const kennan = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function() {
+    return "Hello, my name is Kennan!";
+  }
+};
+const keven = {
+  id: 3,
+  name: "Keven",
+  email: "kmummery2@wikimedia.org",
+  gender: "M"
+};
+const gannie = {
+  id: 4,
+  name: "Gannie",
+  email: "gmartinson3@illinois.edu",
+  gender: "M"
+};
+const antonietta = {
+  id: 5,
+  name: "Antonietta",
+  email: "adaine5@samsung.com",
+  gender: "F",
+  multiplyNums: function(num1, num2) {
+    let product = num1 * num2;
+    return product;
+  }
+};
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.table(mitzi.name);
```

Comment:
Great use of `console.table` to provide an elegant way of logging results to the console.